static-assets/components/cstudio-forms/controls/image-picker.js

```diff
@@ -871,7 +877,8 @@ YAHOO.extend(CStudioForms.Controls.ImagePicker, CStudioForms.CStudioFormField, {
         name: 'imageManager',
         type: 'datasource:image'
       },
-      { label: CMgs.format(langBundle, 'readonly'), name: 'readonly', type: 'boolean' }
+      { label: CMgs.format(langBundle, 'readonly'), name: 'readonly', type: 'boolean' },
+      { label: CMgs.format(langBundle, 'allowSVG'), name: 'allowSVG', type: 'boolean', defaultValue: 'true' }
```

Comment:
Disallow by default. Should the defaultValue here be a string or a boolean?
```suggestion
      { label: CMgs.format(langBundle, 'allowSVG'), name: 'allowSVG', type: 'boolean', defaultValue: 'false' }
```

Reply:
Even though the value is type boolean it is set as a string and the content-type engine handles/parses it as a boolean